docs/design/specs/Ecma-335-Augments.md

```diff
@@ -1102,3 +1104,19 @@ The incorrect description of `System.Array.Initialize` API in section "II.13.2 I
 ## Debug Interchange Format
 
 The Debug Interchange Format described in partition V is superseded by the [Portable PDB Format](PortablePdb-Metadata.md).
+
+## Instance construction
+
+The following is added to the section "II.10.5.1 Instance constructor":
+
+> Instance constructors shall not be executed multiple times for a single object instance. Explicit calls to constructors on object instances from user code are only permitted when calling instance constructors of the base type inside of instance constructors of the derived type.
```

Comment:
This is not correct. Try this:
```csharp
class A
{
    A() : this(1)
    {
    }

    A(int x)
    {
    }
}
```

Reply:
Also, you may want to update the wording in the `call` instruction verification rules that have the same mistake: "The call instruction can also be used to call an object’s base class constructor, or to initialize a value type location by calling an appropriate constructor, both of which are treated as special cases by verification.". 

It may be sufficient to just fix the `call` verification rules to be more precise instead of duplicating it here.